widget-missing-in-kendoui-professional-package.md

```diff
@@ -0,0 +1,97 @@
+
+---
+title: Widgets from the Kendo UI Core/Professional Package Don't Load
```

Comment:
You can safely go with something like "the Kendo UI Packages" instead of the "the Kendo UI Core/Professional Package"